internal/pkg/network/setup.go

```diff
@@ -102,21 +102,22 @@ func ifup(ifname string, mtu int) (err error) {
 	case netlink.OperUnknown:
 		fallthrough
 	case netlink.OperDown:
-		if mtu > 0 {
-			if err = netlink.LinkSetMTU(link, mtu); err != nil {
-				return err
-			}
-		}
 		if err = netlink.LinkSetUp(link); err != nil && err != syscall.EEXIST {
 			log.Printf("im failing here in operdown for %s", ifname)
```

Comment:
lol on the log message here

Reply:
lol, yes that needs to be fixed ... although brad is rewriting `networkd` so it will probably go away then.